build.psm1

```diff
@@ -169,12 +169,12 @@ function Start-PSBuild {
 
     # set output options
     $OptionsArguments = @{
-        Publish=$Publish
-        Output=$Output
-        FullCLR=$FullCLR
-        Runtime=$Runtime
-        Configuration=$Configuration
-        Verbose=$true
+        Publish=$Publish;
```

Comment:
I suppose we don't have a style guideline here - but `;` isn't necessary, a newline is sufficient. I prefer just a newline unless the hashtable is all on one line.

Note that if your goal is consistency, there is at least 1 other multi-line hashtable that doesn't use `;`.